package_bag/routines.py

```diff
@@ -72,15 +73,13 @@ def unpack_rename(self, bag_path, tmp):
     def validate_metadata(self, bag_path):
         """Validates the bag-info.txt file against the bagit profile"""
         new_bag = bagit.Bag(bag_path)
-        if "BagIt-Profile-Identifier" not in new_bag.info:
-            raise TypeError("No BagIt Profile to validate against")
+        with open("/code/package_bag/zorya_bagit_profile.json", "r") as fp:
+            data = json.load(fp)
+        profile = bagit_profile.Profile("zorya_bagit_profile.json", profile=data)
```

Comment:
If I'm understanding correctly, the `zorya_bagit_profile.json` is an important value because it is the filename of the profile AND ALSO has to be passed in as the profile identifier. If that's true, then I'd suggest considering setting a variable or even a config for this value.